frontend/app/src/components/defi/display/LendingAssetTable.vue

```diff
@@ -14,7 +14,19 @@
       </span>
     </template>
     <template #item.protocol="{ item }">
-      <defi-protocol-icon :protocol="item.protocol" />
+      <div class="d-flex flex-row lending-asset-table__protocol__">
+        <defi-protocol-icon
+          :protocol="item.protocol"
+          class="lending-asset-table__protocol__icon"
+        />
+        <span class="lending-asset-table__protocol__name align-self-center">
+          {{
+            item.protocol === 'makerdao'
+              ? 'MakerDAO DSR'
+              : item.protocol | capitalize
+          }}
+        </span>
+      </div>
```

Comment:
Is there any specific reason why we are not adding the label in the `DefiProtocolIcon` component? This way this label is only picked by the LendingAssetTable but if we include it in the component it will also be available to premium components.

Reply:
I found it like this =D 

Maybe in some cases it's worthwhile to have the DefiProtocolIcon separate anyway (we might want to just use the icon somewhere) so I can make a DefiLabel component which is the label + the icon?

Or I can add a flag to the Icon component to render: label only / icon only / label + icon?

Reply:
:D the label + flag idea sounds great :) If we can make it display both by default that would be nice. We only cannot easily change the name because the DefiProtocolIcon is also used in the premium components, and we would have to introduce some breaking changes.

Reply:
If I add this in this PR then in the premium components we need to remove the label there as it'll show up double right?